Update LavaMoat policies

mcmire opened the pull request:
It appears that following recent updates to dependencies the LavaMoat
policies are out of date. This commit fixes that.

darkwing commented:
Is there any context that can be added here on GitHub or via DM?

Gudahtt commented:
The context is that somebody changed a dependency and forgot to run `yarn lavamoat:auto`. Though from the look of those conflicts, a recent merge may have fixed this already.

mcmire committed:
Update LavaMoat policies

It appears that following recent updates to dependencies the LavaMoat
policies are out of date. This commit fixes that.

mcmire commented:
@darkwing Yes, that's all it is, sorry for the lack of description on this.

@Gudahtt I've regenerated the policies based on the latest changes.

Gudahtt reviewed (dismissed):
LGTM!

Gudahtt commented:
Hmm. CI e2e tests are showing a blank screen crash.

mcmire commented:
Urgh, okay, I'll take a look.

mcmire committed:
Update policies after unlinking controllers

Gudahtt reviewed (approved):
LGTM!
